docs/HowTo/Find-and-Connect/Managing-Peers.md

```diff
@@ -2,72 +2,91 @@
 description: Managing Hyperledger Besu peers
 ---
 
-# Managing peers
+# Manage peers
 
-## Limiting peers
+Hyperledger Besu peer-to-peer (P2P) discovery happens periodically based on the number of peers in a network and the
+node's [peer limit](#limit-peers).
 
-Limiting peers reduces the bandwidth used by Hyperledger Besu. Limiting peers also reduces the CPU
+The frequency of discovery isn't configurable, but you can [limit remote connections](#limit-remote-connections) in
+public networks and [randomly prioritize connections](../../Reference/CLI/CLI-Syntax.md#random-peer-priority-enabled)
+in small, stable networks.
+
+!!! info
+
+    You can use [`admin_addPeer`](../../Reference/API-Methods.md#admin_addpeer) to attempt a specific connection, but
+    this isn't P2P discovery.
+
+We recommend [using bootnodes](Bootnodes.md) to initially discover peers.
+
+## Limit peers
+
+Limiting peers reduces the bandwidth used by Besu. Limiting peers also reduces the CPU
 time and disk access used to manage and respond to peers.
 
 To reduce the maximum number of peers, use the
 [`--max-peers`](../../Reference/CLI/CLI-Syntax.md#max-peers) option. The default is 25.
 
-## No discovery
+## Limit remote connections
 
-To disable P2P discovery, use the
-[`--discovery-enabled`](../../Reference/CLI/CLI-Syntax.md#discovery-enabled) option.
+In public networks, enabling the [remote connection limits](../../Reference/CLI/CLI-Syntax.md#remote-connections-limit-enabled)
```

Comment:
"To prevent eclipse  attacks in public networks when using..."

Reply:
Suggestion: "Prevent eclipse attacks when using [`--sync-mode`](../../Reference/CLI/CLI-Syntax.md#sync-mode) and [`--fast-sync-min-peers`](../../Reference/CLI/CLI-Syntax.md#fast-sync-min-peers) on public networks by enabling the [remote connection limits](../../Reference/CLI/CLI-Syntax.md#remote-connections-limit-enabled)"

[risk to prevent] -> [when risk can happen] + [where risk can happen] -> [fix]